os/arch/arm/src/common/up_reboot_reason.c

```diff
@@ -28,11 +28,11 @@
  * Name: reboot_reason_trywrite
  ****************************************************************************/
 #ifdef CONFIG_SYSTEM_REBOOT_REASON
-void reboot_reason_write_user_intended(void)
+void reboot_reason_write_assert(void)
```

Comment:
How about renaming `reboot_reason_try_write_assert`?
Because this function has the conditional - is written.

Reply:
yes, It is good idea to keep the function name ```reboot_reason_try_write_assert```. Let  me update it.

Thank you for the suggestion.